### PR TITLE
final report stakeholders

### PR DESCRIPTION
**Pull request by jagprog5**

### CODE REVIEW

File: reports/final/report.tex [hunk 1]
```diff
@@ -1542,6 +1542,50 @@ \section{Stakeholder Needs} \label{sec:stakeholder-needs} % John / Tim
...
+%% Ethan, if you could elaborate on why the kits were bad that would be great.
+%% John finds it difficult to comment since he did not use the kits
+%% Emphasis on the "quality" point
+
+%% and to double check, we did indeed have scroll to zoom functionality?
+
+In the online electrical engineering labs during the COVID pandemic, Professor
+Whitehall noticed issues with the kits provided to students. There was a general
+lack of quality and brittleness which caused students to require replacement
+kits. Additionally, the implementation of certain features was poor; the display
```
Comment (thetimmorland) [inline reports/final/report.tex]: "was poor" should be "were poor"

File: reports/final/report.tex [hunk 2]
```diff
@@ -1542,6 +1542,50 @@ \section{Stakeholder Needs} \label{sec:stakeholder-needs} % John / Tim
...
+was unable to simultaneously show the frequency and time domain, and it was
+poorly indicated which was currently being dislpayed.
```
Comment (thetimmorland) [inline reports/final/report.tex]: misspelled displayed

**Review by thetimmorland — APPROVED** [review]
Approved, but see comments regarding spelling/grammar.

File: reports/final/report.tex [hunk 3]
```diff
@@ -1675,19 +1719,20 @@ \section{Overall Team Effort}
 % NOTE: I think we will need to add some further reflection here about how the
 % project went overall, it is in the rubric
 
+% if you would like me to revise this effort table, please let me know
```
Comment (EYangster) [inline reports/final/report.tex]: You should not be editing the effort table at this time.

Comment (EYangster) [reply reports/final/report.tex]: Let Ethan do it at the end.

File: reports/final/report.tex (hunk 1 above)
Comment (jagprog5) [reply reports/final/report.tex]: 8eca5b13e067e762d9c268a3869be6e17a714cce

File: reports/final/report.tex (hunk 2 above)
Comment (jagprog5) [reply reports/final/report.tex]: 4e2850c1f654f0c15314b928c9dab49abc75b077

File: reports/final/report.tex (hunk 3 above)
Comment (jagprog5) [reply reports/final/report.tex]: a88c1ae5a9bae3099c6a182a36564f77b4af1bc1

**Review by EYangster — APPROVED** [review]

**Pull request merged**